### DISCUSSION TITLE
I have set the terminal text cursor to beam (|), but entering nu set it back to block (█), how do I stop this behavior?

### DISCUSSION THREAD

**Discussion by KSXGitHub** [nushell/nushell#7270] [Q&A] [answered]
I have let `nu` generate `config.nu` for me.

**Comment by fdncred** [comment, marked as answer]
This should hopefully fix this situation in the near future. https://github.com/nushell/reedline/pull/515

**Reply by KSXGitHub** [reply]
Should I mark this as "Answered" or not? Because this discussion is resolved, but it's not exactly an answer.

**Reply by fdncred** [reply]
Yes, your question is answered even though the issue is not resolved. Future users to this topic can follow the link to see if the issue is closed. We could also link the issue too for future users. I thought you had an issue but I can't find it now.
